profile/2020-09-01-hybrid.json

```diff
@@ -583,6 +583,34 @@
           "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/ResourceProvider.json"
         }
       ]
+    },
+    "microsoft.containerregistry.admin": {
```

Comment:
This map is only for user facing resource types, not admin

Reply:
updated with user-facing resource types for ACR